packages/mgt/src/components/mgt-get/mgt-get.ts

```diff
@@ -8,6 +8,26 @@
 import { customElement, html, property } from 'lit-element';
 import { Providers, ProviderState, MgtTemplatedComponent, equals } from '@microsoft/mgt-element';
 import { prepScopes } from '../../utils/GraphHelpers';
+import { blobToBase64 } from '../../utils/Utils';
+import { getPhotoForResource, photosCacheEnabled, storePhotoInCache } from '../../graph/graph.photos';
```

Comment:
I noticed not all imports are used here, consider removing the ones that are not necessary.

Reply:
Good catch! Were used in previous implementation before I use the graph.photos service! Now gone!